### ISSUE TITLE
Synthetic Graph Generators

### ISSUE THREAD

**Issue by jegonzal** [amplab/graphx#11]
To help with benchmarking lets create some synthetic graph generators.  The Pregel paper describes a log-normal generator which is relatively easy to implement.

**Comment by dcrankshaw** [comment]
Where should the graph generator code go?

**Comment by rxin** [comment]
How about in the utils package?

**Comment by jegonzal** [comment]
We should also add an RMat graph generator.  See http://www.cs.cmu.edu/~christos/PUBLICATIONS/siam04.pdf for details.

**Comment by jegonzal** [comment]
Did we ever find a location to place this code?

**Comment by jegonzal** [comment]
This github "bug/feature" is getting a little confusing.  So is this issue closed?  As far as I can tell the generator code has been written but possibly not tested?

**Comment by dcrankshaw** [comment]
Yeah it needs to be cleaned up and I need to write tests for it. I'll do that this week.
